easybuild/easyblocks/generic/waf.py

```diff
@@ -52,7 +52,7 @@ def configure_step(self, cmd_prefix=''):
 
         return res.output
 
-    def build_step(self, verbose=False, path=None):
+    def build_step(self, path=None, *args, **kwargs):
```

Comment:
since `path` is not used:
```suggestion
    def build_step(self, *args, **kwargs):